pyproject.toml

```diff
@@ -367,7 +367,6 @@ exclude = [
   "scripts/",
   "sdist/",
   "src/phoenix/evals/",
-  "tests/",
```

Comment:
shuoldn't we keep them separate?

Reply:
type checks don't run in vscode with this line